.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Astar Release Checklist
+about: Checklist to follow before creating a new release
+title: 'Release v_xx Checklist'
+labels:
+assignees: ''
+---
+
+## Required Changes (PR)
+<!---
+All the PRs that should, For Example,
+- [x] #1000
+- [x] #1006 
+-->
```

Comment:
That should?

Also, how should this be filled out? Manually?

Reply:
What's the benefit of adding PRs manually?
Why do we want to do that?

Reply:
We don't add PRs